build.sh

```diff
@@ -69,10 +69,20 @@ if [ x$PULLREQUEST = "xtrue" ]; then
   fi
 fi
 
-# Build steps for SDK libraries:
+# Verify SDK CLI, run unit tests
+_notify_github pending "SDK CLI build running"
+pushd $REPO_ROOT_DIR/cli
+go test ./...
+if [ $? -ne 0 ]; then
+  _notify_github failure "SDK CLI build failed"
```

Comment:
Quick question @jsrodman. At which point are the dependencies for this installed? Testing this locally (in the docker container), I get:
```
./build.sh
[STATUS] build:sdk pending: SDK CLI build running
/build/cli /build
commands.go:12:2: cannot find package "github.com/mesosphere/dcos-commons/cli/client" in any of:
	/usr/lib/go-1.8/src/github.com/mesosphere/dcos-commons/cli/client (from $GOROOT)
	/go-tmp/src/github.com/mesosphere/dcos-commons/cli/client (from $GOPATH)
commands.go:13:2: cannot find package "github.com/mesosphere/dcos-commons/cli/commands" in any of:
	/usr/lib/go-1.8/src/github.com/mesosphere/dcos-commons/cli/commands (from $GOROOT)
	/go-tmp/src/github.com/mesosphere/dcos-commons/cli/commands (from $GOPATH)
commands.go:14:2: cannot find package "github.com/mesosphere/dcos-commons/cli/config" in any of:
	/usr/lib/go-1.8/src/github.com/mesosphere/dcos-commons/cli/config (from $GOROOT)
	/go-tmp/src/github.com/mesosphere/dcos-commons/cli/config (from $GOPATH)
commands.go:15:2: cannot find package "gopkg.in/alecthomas/kingpin.v3-unstable" in any of:
	/usr/lib/go-1.8/src/gopkg.in/alecthomas/kingpin.v3-unstable (from $GOROOT)
	/go-tmp/src/gopkg.in/alecthomas/kingpin.v3-unstable (from $GOPATH)
[STATUS] build:sdk failure: SDK CLI build failed
/build
```